src/routes/(console)/project-[region]-[project]/+layout.ts

```diff
@@ -11,16 +11,20 @@ import { get } from 'svelte/store';
 import { headerAlert } from '$lib/stores/headerAlert';
 import PaymentFailed from '$lib/components/billing/alerts/paymentFailed.svelte';
 import { loadAvailableRegions } from '$routes/(console)/regions';
-import type { Organization } from '$lib/stores/organization';
+import { roles as rolesStore, scopes as scopesStore } from '$lib/stores/roles';
+import { organization as orgStore, type Organization } from '$lib/stores/organization';
 
 export const load: LayoutLoad = async ({ params, depends }) => {
     depends(Dependencies.PROJECT);
     let currentPlan: Plan = null;
+    const orgFromStore: Organization = get(orgStore);
```

Comment:
why do we get org from store? there are race conditions from that are very unreliable. we should only need to fetch it once.

Reply:
to not fetch from api if it already exists.